doc/lua_api.txt

```diff
@@ -3612,7 +3612,7 @@ Definition tables
             max_items = 1,  -- Maximum number of items to drop.
             items = { -- Choose max_items randomly from this list.
                 {
-                    items = {"foo:bar", "baz:frob"},  -- Choose one item randomly from this list.
+                    items = {"foo:bar", "baz:frob"},  -- Items to drop.
                     rarity = 1,  -- Probability of getting is 1 / rarity.
```

Comment:
I'd suggest changing 'getting' to 'dropping' as well for consistency in terminology ?